ci/package.sh

```diff
@@ -59,70 +59,66 @@ do
 
                 if [ $rebuild_local = true ]
                 then
-                    echo -e "\n- BUILDING stack: $repo_name/$stack_id"
-
-                    if [ -d $stack_dir/image ]
-                    then
-                        if ${CI_WAIT_FOR} image_build \
-                            --build-arg GIT_ORG_REPO=$GIT_ORG_REPO \
-                            --build-arg IMAGE_REGISTRY_ORG=$IMAGE_REGISTRY_ORG \
-                            --build-arg STACK_ID=$stack_id \
-                            --build-arg MAJOR_VERSION=$stack_version_major \
-                            --build-arg MINOR_VERSION=$stack_version_minor \
-                            --build-arg PATCH_VERSION=$stack_version_patch \
-                            --label "org.opencontainers.image.created=$(date +%Y-%m-%dT%H:%M:%S%z)" \
-                            --label "org.opencontainers.image.version=${stack_version}" \
-                            --label "org.opencontainers.image.revision=$(git log -1 --pretty=%H)" \
-                            --label "appsody.stack=$IMAGE_REGISTRY_ORG/$stack_id:$stack_version" \
-                            -t $IMAGE_REGISTRY_ORG/$stack_id \
-                            -t $IMAGE_REGISTRY_ORG/$stack_id:$stack_version \
-                            -t $IMAGE_REGISTRY_ORG/$stack_id:$stack_version_major \
-                            -t $IMAGE_REGISTRY_ORG/$stack_id:$stack_version_major.$stack_version_minor \
-                            -f $stack_dir/image/Dockerfile-stack $stack_dir/image \
-                            > ${build_dir}/image.$stack_id.$stack_version.log 2>&1
-                        then
-                            trace  "Output from image build" "${build_dir}/image.$stack_id.$stack_version.log"
-
-                            echo "created $IMAGE_REGISTRY_ORG/$stack_id:$stack_version"
-                            echo "$IMAGE_REGISTRY_ORG/$stack_id" >> $build_dir/image_list
-                            echo "$IMAGE_REGISTRY_ORG/$stack_id:$stack_version" >> $build_dir/image_list
-                            echo "$IMAGE_REGISTRY_ORG/$stack_id:$stack_version_major" >> $build_dir/image_list
-                            echo "$IMAGE_REGISTRY_ORG/$stack_id:$stack_version_major.$stack_version_minor" >> $build_dir/image_list
-                        else
-                            echo "Error building $IMAGE_REGISTRY_ORG/$stack_id:$stack_version"
-                            cat ${build_dir}/image.$stack_id.$stack_version.log
-                            exit 1
+                    appsody_cmd="appsody stack validate"
```

Comment:
Why not move this in the else block to keep it closer to the `echo`

Reply:
Done